### ISSUE TITLE
Commando packages: incorrect context menu

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#896]
- open commando
- go to packages scope
- select a package and press enter

*Open* and *favorite* shouldn't  be in that menu
If already installed, *disable* should be an option.

**Comment by Naatan** [comment]
Open/Favourite should only happen on certain sections. Were you searching straight from the packages scope or from a sub-menu?

Enable/disable is currently not supported.

**Comment by cgchoffman** [comment]
I was in "addons" mode.  I'm not sure where that opens to; Tools > Addons: select one of the installed packages and you can "favourite" it or "open" it which does nothing.  

Disable should be added.

**Comment by Naatan** [comment]
Right then that's the right behavior. You should have also seen an Uninstall option, no?

**Comment by cgchoffman** [comment]
Yes.  The packages specific menu options are right but open and favorite shouldn't be these.

**Comment by Naatan** [comment]
They occur on any expanded Commando item. Open just executes the item and favourite, well that's self explanatory. They are there by design.

**Comment by cgchoffman** [comment]
Ok `Favourite` makes sense for what it's used for and it is working.  I thought it wasn't.  `Open` doesn't do anything so I think this should be re-thought.  It should either be removed or we need to add the functionality to open tool if it makes sense.  Eg, DOM-Inspector should open it, Abyss Skin (any skin) open the prefs(?), etc.

**Comment by Naatan** [comment]
Right, in the current context the action of Open is the very menu you are in, so it doesn't make sense. I like your ideas on what to do instead, but given our current package structure I don't think it's possible to infer the necessary information to perform such actions. So instead we should just not show "Open" when it is not relevant in the current context.
